src/app/css/_deviceEvents.scss

```diff
@@ -37,10 +37,33 @@
         @include themify($themes) {
             color: themed('textColor');
         }
-        width: 500px;
+        width: 80%;
+    }
+
+    .custom-event-hub-label {
+        font-weight: 600;
+        font-size: 14px;
+        @include themify($themes) {
+            color: themed('textColor');
+        }
+    }
+
+    .custom-event-hub-text-field {
+        display: grid;
+        padding-left: 24px;
+        padding-bottom: 10px;
+        @include themify($themes) {
+            color: themed('textColor');
+        }
+        width: 80%;
     }
 
     .toggle-button {
         padding-left: 24px;
     }
+
+    .scrollable-telemetry {
+        height: calc(100vh - 400px);
```

Comment:
Suggestion - for items with calculated dimensions that use a hard-coded number value of pixels (also see _devicePnpDetailList.scss), we might want to store these in variables to be used elsewhere, or at least so we know why they're offset by 400/360px

Reply:
I agree. I think we really need a better to handle these all together. We have a lot of inline scrolls, I was wondering if we can a more automatically way to handle these.